apps/sms/views/shared/js/activity_handler.js

```diff
@@ -71,12 +71,17 @@ var ActivityHandler = {
       // It's reasonable to focus on composer if we already have some phone
       // number or even contact to fill recipients input
       focusComposer = true;
-      // try to get a thread from number
-      // if no thread, promise is rejected and we try to find a contact
-      threadPromise = MessageManager.findThreadFromNumber(viewInfo.number)
-        .then((threadId) => viewInfo.threadId = threadId)
-        // case no contact and no thread id: gobble the error
-        .catch(() => {});
+
+      if (!viewInfo.body) {
```

Comment:
__nit:__ maybe add comment explaining why we decided to not search for conversation if body is presented? I think it's easy to forget, okay at least for me :)